examples/HighchartsGanttInvertedChart/App.js

```diff
@@ -0,0 +1,86 @@
+import React, {Component} from 'react';
+import {
+  GanttSeries,
+  HighchartsGanttChart,
+  Subtitle,
+  Title,
+  Tooltip,
+  withHighcharts,
+  XAxis,
+  YAxis
+} from 'react-jsx-highcharts-gantt';
+import ExampleCode from '../utils/ExampleCode';
+import code from './exampleCode';
+
+
+class App extends Component {
+
+  render() {
+
+    return (
+      <div className="app">
+        <HighchartsGanttChart
+          inverted={true}
+          plotBorderWidth={1}
+        >
+          <Title>Inverted Gantt Chart</Title>
+          <XAxis>
+            {
+              [
+                {
+                  opposite: false
+                },
+                {
+                  opposite: false
+                }
+              ]
+            }
```

Comment:
This isn't how React JSX Highcharts works. This should be `<XAxis opposite={false}>`.

Plus we've only allowed for a single XAxis at the moment, so we need to revisit the XAxis component.

Reply:
OK. Thank for the input. I felt I was doing something wrong. It was this original Higcharts code that confused me:
```
xAxis: [{
        opposite: false
    }, {
        opposite: false
}]
```